content/en/blog/_posts/2021-08-13-server-side-apply-ga.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: blog
+title: "Kubernetes 1.22: Server Side Apply moves to GA"
+date: 2021-08-13
+slug: server-side-apply-ga
+---
+
+**Authors:** Jeffrey Ying, Google & Joe Betz, Google
+
+Server-side Apply (SSA) has been promoted to GA in the Kubernetes v1.22 release. The GA milestone means you can depend on the feature and its API, without fear of future backwards-incompatible changes. GA features are protected by the Kubernetes [deprecation policy](/docs/reference/using-api/deprecation-policy/).
+
+## What is Server-side Apply?
+
+Server-side Apply helps users and controllers manage their resources through declarative configurations. Server-side Apply replaces the client side apply feature implemented by “kubectl apply” with a server-side implementation, permitting use by tools/clients other than kubectl. Server-side Apply is a new merging algorithm, as well as tracking of field ownership, running on the Kubernetes api-server. Server-side Apply enables new features like conflict detection, so the system knows when two actors are trying to edit the same field. Refer to the [Server-side Apply Documentation](https://kubernetes.io/docs/reference/using-api/server-side-apply/) and [Beta 2 release announcement](https://kubernetes.io/blog/2020/04/01/kubernetes-1.18-feature-server-side-apply-beta-2/) for more information.
```

Comment:
Change URL so it appears as: `[Server-side Apply Documentation](/docs/reference/using-api/server-side-apply/)`